### PR TITLE
DD documentation

### PR DESCRIPTION
**Pull request by jinder1s**
PR for creating documentation for DD relating specifically to enterprise-catalog.  Please comment if you think something else should be added.

### CODE REVIEW

File: docs/decentralized_devstack_workflows.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+----------------------------
+
+1. Clone the enterprise-catalog repo and **cd into that directory**
+2. Create and activate either python 3.5 or 3.8 virtual environment, for more info: `virtualenv`_
+3. install requirements by running: `make requirements`
+3. open .env file and uncomment the line `COMPOSE_FILE=decentralized_devstack/docker-compose.yml` by removing "#" symbol
+4. provision Decentralized devstack by running: `./decentralized_devstack/provision.sh`
+5. Once provisioning has successfully run, you can view the enterprise catalog at http://localhost:18160/admin
+
+  - You can login with the username *edx* and password *edx*.
```
Comment (jinder1s) [inline docs/decentralized_devstack_workflows.rst]: I can't seem to format this sublist correctly, any suggestions?

Comment (jinder1s) [reply docs/decentralized_devstack_workflows.rst]: Figured it out!

File: docs/decentralized_devstack_workflows.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+.. role:: bash(code)
+   :language: bash
+
+Getting started from scratch
+----------------------------
+
+#. Clone the enterprise-catalog repo and **cd into that directory**
+#. Create and activate either python 3.5 or 3.8 virtual environment, for more info: `virtualenv`_
+#. install requirements by running: `make requirements`
```
Comment (feanil) [inline docs/decentralized_devstack_workflows.rst]:
```suggestion
#. Run `make requirements` to install requirements.
```

File: docs/decentralized_devstack_workflows.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+#. open .env file and uncomment the line :bash:`COMPOSE_FILE=decentralized_devstack/docker-compose.yml` by removing "#" symbol
```
Comment (feanil) [inline docs/decentralized_devstack_workflows.rst]:
```suggestion
#. Open `.env` file and uncomment the line :bash:`COMPOSE_FILE=decentralized_devstack/docker-compose.yml` by removing "#" symbol
```

File: docs/decentralized_devstack_workflows.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+#. provision Decentralized devstack by running: :bash:`$ ./decentralized_devstack/provision.sh`
```
Comment (feanil) [inline docs/decentralized_devstack_workflows.rst:16]:
```suggestion
#. Provision decentralized devstack by running: :bash:`$ ./decentralized_devstack/provision.sh`
```

**Review by feanil — APPROVED** [review]
Generally looks good.

One formatting issue, you're sometimes capitalizing the first word in a bulleted/numbered items and sometimes not.  It also seems like some of these list of instructions could just be the list of commands to run which might be easier to find and use?

**Review by feanil — APPROVED** [review]

**Pull request merged**